.github/workflows/verify.yml

```diff
@@ -0,0 +1,96 @@
+name: Verify
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    services:
+      db:
+        image: postgres:14
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - "5432:5432"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1.115.3
+        with:
+          ruby-version: 3.1.2
+
+      - name: Install Dependencies
+        run: |
+          sudo apt install -yqq libpq-dev
+          gem install bundler
+
+      - name: Install Gems
+        run: |
+          bundle install
+
+  # We don't have RSpec files for testing:
+  #      - name: Run Tests
+  #        run: |
+  #          bundle exec rspec
+
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.115.3
+        with:
+          ruby-version: 3.1.2
+
+      - name: Install Brakeman
+        run: |
+          gem install brakeman
+
+      - name: Run Brakeman
+        run: |
+          brakeman -f json > tmp/brakeman.json || exit 0
+
+      - name: Brakeman Report
+        uses: devmasx/brakeman-linter-action@v1.0.0
+        env:
+          REPORT_PATH: tmp/brakeman.json
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+  linters:
+    name: linters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      #       - name: rubocop
+      #         uses: reviewdog/action-rubocop@v1
+      #         with:
+      #           rubocop_version: gemfile
+      #           rubocop_extensions: rubocop-rails:gemfile rubocop-rspec:gemfile
+      #           github_token: ${{ secrets.github_token }}
+
+      - name: brakeman
+        uses: reviewdog/action-brakeman@v1
+        with:
+          brakeman_version: gemfile
+          github_token: ${{ secrets.github_token }}
+
+      - name: reek
+        uses: reviewdog/action-reek@v1
+        with:
+          reek_version: gemfile
+          github_token: ${{ secrets.github_token }}
+
+      - name: fasterer
+        uses: vk26/action-fasterer@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```

Comment:
Please, add empty line on the end of the file.

Reply:
Update